tests/tests_signin_user.py

```diff
@@ -0,0 +1,7 @@
+def test_with_authenticated_client(client, django_user_model):
+    username = "test"
+    password = "test"
+    django_user_model.objects.create_user(username=username, password=password)
+    client.force_login(user)
+    respons = client.get('/')
+    assert respons.status_code == 200
```

Comment:
I'm pretty sure that if the user was not logged in it would still get a 200 when going to the main page ('/'). Probably what you want to do is send it to a page that requires login, and gives an error when not logged in.

Reply:
I think I did not understand you so much
You can explain more

Reply:
@BarBenNaim I will explain,
What is happening when you get to '/' It will reply with 200 since the main page does not require login.
Assume you are pressing login, you have to go to /sign_in, over there you should be either logged in or logged out,
You can verify the content of the page, or redirected to personal area, you need to check that flow, also

Reply:
OK, but if I change to: respons = client.get('/sign_in') ill get status code 301
because I move to personal_area. so, how can I check that flow?

Reply:
can i do somthing like this?
def test_with_authenticated_client(client, django_user_model):
	username = "test"
	password = "test"
	user = django_user_model.objects.create_user(username=username, password=password)
	client.force_login(user)
	respons1 = client.get('/sign_in')
	respons2 = client.get('/personal_area')
	assert respons1.status_code == 301 and respons2.status_code == 200

Reply:
that would work.
What happens when you go to sign_in before the login?
You can add some code showing that before login you get a 200 on sign_in and a 511 in personal_area
and after login you get a 301 and 200

Reply:
After PR #96  is merged, it will actually be sign_in gives a 200 and personal_area gives a 301 before login in.